code/modules/organs/organ.dm

```diff
@@ -28,16 +28,18 @@
 	var/min_broken_damage = 30             // Damage before becoming broken
 	var/max_damage = 30                    // Damage cap
 	var/rejecting                          // Is this organ already being rejected?
-	var/death_time                         // world.time at moment of death.
+	var/death_time                         // REALTIMEOFDAY at moment of death.
 	var/scale_max_damage_to_species_health // Whether or not we should scale the damage values of this organ to the owner species.
 
 /obj/item/organ/Destroy()
 	if(owner)
-		owner.remove_organ(src, FALSE, FALSE, TRUE, TRUE, FALSE) //Tell our parent we're unisntalling silently
+		owner.remove_organ(src, FALSE, FALSE, TRUE, TRUE, FALSE) //Tell our parent we're unisntalling in place
 	else
 		do_uninstall(TRUE, FALSE, FALSE, FALSE) //Don't ignore children here since we might own/contain them
 	owner = null
 	dna = null
+	species = null
```

Comment:
Why are these being nulled?

Reply:
Just because its a reference. Even if species are singletons, I got the reflex to clear any dangling refs.

Reply:
It's a bit untidy, and in this case it's not a circular ref so it's unnecessary for GC. Not a huge issue.